neuralmonkey/config/configuration.py

```diff
@@ -67,11 +63,40 @@ def load_file(self, path):
             log("Failed to load INI file: {}".format(exc), color='red')
```

Comment:
Když to napíše `Loading INI file` a pak hodí error `Value of field ... does not satisfy condition ...`, tak neni potřeba nic odvozovat. Každopádně - je lepší to zahodit  a do tý jediný exception, která se tam vyhazuje přidat na začátek `Error in INI file: Value of field ...`